package.json

```diff
@@ -5,8 +5,11 @@
   "main": "index.js",
   "scripts": {
     "lint": "standard",
-    "pretest": "npm run lint",
-    "test": "grunt test"
+    "integration-test": "npx karma start examples/simple/karma.conf.js --single-run --browsers ChromeHeadless",
+    "unit-test": "npx mocha test/*",
+    "test": "npm run integration-test && npm run unit-test",
```

Comment:
nit: I'm not very familiar with this repo, but maybe we should run unit tests before the integration tests?

Reply:
True! Will fix.